### PR TITLE
Show big red warning on Demo Site so folks don't think it's the real site

### LINKED ISSUE #1133: Disabling Old Insights Link But Still Google-Able
Tracked down the source of a few users who haven't been able to log in in the last couple of weeks.  Instead of going to Favorites or using our link, they Google "Student Insights" and come up with the old site that looks just the current site:

https://somerville-teacher-tool-demo.herokuapp.com/

Can we disable this?

### PR DESCRIPTION
**Pull request by alexsoble**
# Issue 

+ Close #1133.

# Screenshot (with warning)

screen shot 2017-10-05 at 2 04 01 pm

# Screenshot (without warning)

screen shot 2017-10-05 at 2 05 08 pm

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,6 +11,12 @@
   </head>
   <body class="<%= controller_name %> <%= action_name %> <%= if educator_signed_in? then "educator-signed-in" end %>">
     <div class="nav">
+      <% if ENV['DEMO_SITE_WARNING'] === 'SHOW' %>
```
Comment (kevinrobinson) [inline app/views/layouts/application.html.erb]: Alternately I think there's a `DEPLOYMENT_KEY` value or something like that already set that's used to indicate that it's the demo site.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -11,6 +11,12 @@
...
+        <div class="big_warning">
+          <div>This is the demo site! It's filled with fake data.</div>
+          <div>The real site is <a>somerville.studentinsights.org</a>.</div>
```
Comment (kevinrobinson) [inline app/views/layouts/application.html.erb]: Does this need an href?

Comment (alexsoble) [reply app/views/layouts/application.html.erb]: Ya! Good catch. :D

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (alexsoble) [reply app/views/layouts/application.html.erb]: Ooooooh! I thought we had something like this but I couldn't remember what it was called! Thank you @kevinrobinson!

**Comment by alexsoble** [comment]
OK @snoopyuri @kevinrobinson the big red warning is now live at https://somerville-teacher-tool-demo.herokuapp.com/.

**Comment by alexsoble** [comment]
Big and red and hard to miss.

**Pull request merged**